luigi/rpc.py

```diff
@@ -71,7 +72,7 @@ def _async_fetch(self, full_url, body):
 
     def _fetch(self, url, body, log_exceptions=True, attempts=3):
 
-        full_url = 'http://%s:%d%s' % (self._host, self._port, url)
+        full_url = 'http://%s:%d%s%s' % (self._host, self._port, self._url_prefix, url)
```

Comment:
It's what I meant by pyformat (which is in fact something [else](https://www.python.org/dev/peps/pep-0249/#paramstyle)).

(They call it [old %-formatting](https://docs.python.org/3.4/library/string.html#format-examples), but I don't see it going away any time soon.)